### PR TITLE
[PHR-675] | Dheeraj | status API contract

### PR DESCRIPTION
**Pull request by dheeraj-p**

### CODE REVIEW

File: src/main/resources/static/swagger.yaml [hunk 1]
```diff
@@ -272,6 +272,27 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/PatientHealthDataResponse'
+  /v1/cm/patient/health-information/status/:
+    post:
+      tags:
+        - cm app
+      summary: Get health information status.
+      description: This API can be used to checkt the status of health information against a set of data request ids when CM app request for data.
```
Comment (leelanakka) [inline src/main/resources/static/swagger.yaml:280]: spelling of check

File: src/main/resources/static/swagger.yaml [hunk 2]
```diff
@@ -272,6 +272,27 @@ paths:
...
+      parameters:
+        - $ref: '#/components/parameters/authorization'
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/PatientHealthDataFetchRequest'
```
Comment (leelanakka) [inline src/main/resources/static/swagger.yaml:288]: the request body of PatientHealthDataFetchRequest is already there?

File: src/main/resources/static/swagger.yaml (hunk 1 above)
Comment (leelanakka) [inline src/main/resources/static/swagger.yaml]: spelling of check

File: src/main/resources/static/swagger.yaml [hunk 3]
```diff
@@ -249,7 +249,7 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/HealthInformation'
-  /patient/health-information/fetch/:
+  /v1/cm/patient/health-information/fetch/:
```
Comment (angshu) [inline src/main/resources/static/swagger.yaml]: do we need /v1/cm/ prefix? why not just /v1/patient/

Comment (dheeraj-p) [reply src/main/resources/static/swagger.yaml]: We are using same for all other CM App facing APIs. We can omit the `/cm`

File: src/main/resources/static/swagger.yaml (hunk 1 above)
Comment (dheeraj-p) [reply src/main/resources/static/swagger.yaml]: Thanks!

File: src/main/resources/static/swagger.yaml (hunk 2 above)
Comment (dheeraj-p) [reply src/main/resources/static/swagger.yaml:288]: yeah, It is same as the other. If you want we can separate it out

Comment (leelanakka) [reply src/main/resources/static/swagger.yaml:288]: No, That's ok!

**Review by leelanakka — APPROVED** [review]

**Pull request merged**